typing_extensions/src/test_typing_extensions.py

```diff
@@ -2425,7 +2437,10 @@ def test_basic_plain(self):
 
     def test_repr(self):
         Ts = TypeVarTuple('Ts')
-        self.assertEqual(repr(Unpack[Ts]), 'typing_extensions.Unpack[Ts]')
+        if TYPING_3_11_0:
+            self.assertEqual(repr(Unpack[Ts]), '*Ts')
+        else:
+            self.assertEqual(repr(Unpack[Ts]), 'typing_extensions.Unpack[Ts]')
```

Comment:
(Which I forgot to save.)

Wouldn't it make more sense to make the typing_extensions version match the one from Python 3.11?

Reply:
Yes, I was thinking about that too. On the other hand, `*Ts` is a syntax error before 3.11, so it may not be very helpful as `repr()` output.

Reply:
@mrahtz what do you think? Should we use `*Ts` in the repr() for Unpack even before 3.11?

Reply:
I think I agree it should be `Unpack[Ts]` before 3.11 - isn't the `repr()` suppose to be as close as possible to something you can directly `eval()`?

Reply:
Thanks, then let's stick with the current behavior.